Require signatures for prepackaged plugins from v10.11 onward (#8241)
mattermost/docs: source/administration-guide/configure/plugins-configuration-settings.rst

+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+----------------------------------------------------------------------------------------+

.. note::

>   - **Mattermost server v10.11 and later**: Pre-packaged plugins require signature validation on startup. Distributions that bundle custom pre-packaged plugins must configure custom public keys via ``PluginSettings.SignaturePublicKeyFiles`` to validate their signatures.

Author: ```suggestion
  From Mattermost server v10.11, pre-packaged plugins require signature validation on startup. Distributions that bundle custom pre-packaged plugins must configure custom public keys via ``PluginSettings.SignaturePublicKeyFiles`` to validate their signatures.
```

|                                                                                                                                                                                 | - Environment variable: ``MM_PLUGINSETTINGS_AUTOMATICPREPACKAGEDPLUGINS``                  |
+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+--------------------------------------------------------------------------------------------+

.. note::

>   **Prepackaged Plugin Installation Behavior**: When system administrators drop plugin files (with their ``.sig`` signature files) into the ``prepackaged_plugins`` directory, the plugins won't install automatically. Prepackaging makes the plugin available for "offline" installation. The plugin will automatically install only when the system administrator pre-configures the ``config.json`` with that plugin enabled.

Author: ```suggestion
  **Prepackaged Plugin Installation Behavior**: When system administrators drop plugin files (with their ``.sig`` signature files) into the ``prepackaged_plugins`` directory, the plugins won't install automatically. Prepackaging makes the plugin available for "offline" installation. The plugin will automatically install only when a system admin pre-configures the ``config.json`` with that plugin enabled.
```
